aws-out-of-cluster.md

```diff
@@ -37,14 +37,11 @@ Tags may take several hours to show up in the *Cost Allocations Tags* section de
 Tags that contain `:` in the key may be converted to `_` in the Kubecost UI due to Prometheus readability. To use AWS Label Mapping Configs, use this mapping format:
 
 ```
-        helm:
-          parameters:
-            - name: kubecostProductConfigs.labelMappingConfigs.enabled
-              value: 'true'
-            - name: kubecostProductConfigs.labelMappingConfigs.namespace_external_label
-              value: 'mycompanybilling_namespace'
-            - name: kubecostProductConfigs.labelMappingConfigs.cluster_external_label
-              value: 'mycompanybilling_cluster'
+kubecostProdutConfigs:
```

Comment:
Is this a spelling error? Produt

Reply:
@matt-land Yes! Good catch. My fault for the misspell.

@bstuder99 Could you do another PR to correct the misspell?